cypress-io/cypress
# chore: update debug artifact order to display replay first

Update the debug artifact display order to match pattern

### Additional details
- Replay is now the first entry elsewhere in the app, update `getDebugArtifacts` to match

### Steps to test
1. Create/link to repository with failing tests
2. Navigate to Debug page
3. Verify Presence of "View Replay" button on test failures is the first element

### How has the user experience changed?

![Screen Shot 2023-08-08 at 11 13 49 AM (2)](https://github.com/cypress-io/cypress/assets/22619500/197ef82c-62de-4530-b12d-3c2c5b6deaab)

### PR Tasks


- [n/a] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [n/a] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
